packages/velo-external-db-core/lib/router.js

```diff
@@ -17,8 +19,14 @@ const { Find: FIND, Insert: INSERT, BulkInsert: BULK_INSERT, Update: UPDATE, Bul
 
 let schemaService, operationService, externalDbConfigClient, schemaAwareDataService, cfg, filterTransformer, aggregationTransformer, roleAuthorizationService, dataHooks, schemaHooks
 
-let appInfoEnabled = false
-const enableAppInfo = () => appInfoEnabled = true
+let appInfoTemplate
+
+const getAppInfoTemplate = async() => {
```

Comment:
I dont think that this is the right place for the function, lets move it to a util file